File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, padword) => {
//for loop to iterate through array and add to reach minsize
    let finalArr = []
    if(padword != undefined){
        while (finalArr.length < minSize){
            finalArr.push(padword)
            console.log(finalArr)
        }
    }else{
        while (finalArr.length < minSize){
            finalArr.push(null)
            console.log(finalArr)  
        }
    }
    for(let i = 0; i < array.length; i++){
        if(array[i] != finalArr[i]){
            finalArr[i] = array[i]
        }
    }
    return finalArr        
}

console.log(pad([1,2,3,4,5,6,7,8,9,10], 30, 'cat'))

File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
from secrets import randbelow
from typing import final


def pad(array, min_size, value = None):
    final_List = []
    
    if value == None:
        while len(final_List) != min_size:
                final_List.append(None)
    else:
        while len(final_List) != min_size:
                final_List.append(value) 
    for i in range(0,len(array)):
        if array[i] != final_List[i]:
            final_List[i] = array[i]  

    return final_List      

print(pad([1,2,3,4,5],14,"frog"))
